android/engine/src/main/java/org/smartregister/fhircore/engine/util/SharedPrefConstants.kt

```diff
@@ -19,3 +19,5 @@ package org.smartregister.fhircore.engine.util
 const val LAST_SYNC_TIMESTAMP = "last_sync_timestamp"
 const val USER_INFO_SHARED_PREFERENCE_KEY = "USER_INFO"
 const val APP_ID_CONFIG = "app_id_config"
+const val OTP_PIN = "otp_pin"
+const val FORCE_LOGIN_VIA_USERNAME = "FORCE_LOGIN_VIA_USERNAME"
```

Comment:
Should have a standard of naming these constants? either lowercase or uppercase

Reply:
updated